example/docker-compose.yml

```diff
@@ -33,4 +33,6 @@ services:
     environment:
       KAFKAPROXY_HOSTNAME: localhost
       KAFKAPROXY_BASE_PORT: 4000
-      KAFKAPROXY_BOOTSTRAP_SERVER: kafka1:9092
\ No newline at end of file
+      KAFKAPROXY_BOOTSTRAP_SERVER: kafka1:9092
+# Uncomment to get trace logging      
```

Comment:
Please remove this comment (not true anymore anyway?)

Reply:
done